runtime/pretty/print.go

```diff
@@ -257,14 +257,8 @@ func (p ErrorPrettyPrinter) writeCodeExcerpts(
 		lineNumberLength := 0
 		if excerpt.startPos != nil {
 
-			plainLineNumberString := strconv.Itoa(excerpt.startPos.Line)
+			plainLineNumberString := strconv.Itoa(excerpt.endPos.Line)
```

Comment:
Why did this change from start to end position?

Reply:
This is used to compute the "length" (number of digits) in the line number string so we know how many spaces we need to allocate for the left side of the `|` in the error excerpt. In the case where number of digits increases, e.g. in an error between lines 8-12, we need to allocate space for the largest line number in the excerpt, which will always be the `endPos`.

Reply:
Thank you for the explanation, that makes sense! Could you please add a comment?